package.json

```
@@ -24,6 +24,7 @@
     "watch": "grunt watch",
     "env:start": "wp-env start && wp-env run cli wp option update permalink_structure '/%postname%/'",
     "env:stop": "wp-env stop",
-    "lint:js": "wp-scripts lint-js"
+    "lint:js": "wp-scripts lint-js",
+    "lint:js-fix": "wp-scripts lint-js --fix"
```

```suggestion
    "lint-js": "wp-scripts lint-js",
    "lint-js:fix": "wp-scripts lint-js --fix"
```

What do you think about this syntax? Might be more common when looking at wp-scripts.

---

Currently, we have these three commands with `:` in the package.json file:

```
    "env:start": "wp-env start && wp-env run cli wp option update permalink_structure '/%postname%/'",
    "env:stop": "wp-env stop",
    "lint:js": "wp-scripts lint-js"
```

The `env` commands use the `:`, so I think we should maintain these commands or change these three and the new one.